ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__multi-nightly-4.13-upgrade-from-stable-4.12.yaml

```diff
@@ -83,12 +74,25 @@ tests:
     - ref: ipi-install-heterogeneous
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi
+- as: aws-ipi-arm64-p2-f14
+  cron: 9 14 5,20 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      ADDITIONAL_WORKER_ARCHITECTURE: x86_64
```

Comment:
@aleskandro Could you please help confirm if it's needed and if the value is correct? Thank you!

Reply:
Yes, I'd keep it explicitly set and when we start from arm64 clusters, it's correct to set additional workers with x86_64 arch.

Reply:
You also need to set

`ADDITIONAL_WORKER_VM_TYPE: m5.xlarge` (is this the default one we're using for x86 machines right?)

Reply:
It's m6i.xlarge by default in 4.13. But m5.xlarge is supported as well.

Reply:
Ok to use m6i.xlarge as well

Reply:
Updated. Thanks!